packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -0,0 +1,238 @@
+            buttonLabel="Find"
+            onApply={onApplyClicked}
+            onReset={onResetClicked}
+          />
+        )}
+      </div>
+      <div className={crudBarStyles}>
+        <div className={toolbarLeftActionStyles}>
+          {!readonly && (
+            <AddDataMenu
+              insertDataHandler={insertDataHandler}
+              isWritable={isWritable}
+              instanceDescription={instanceDescription}
+            />
+          )}
+          <Button
+            leftGlyph={<Icon glyph="Export" />}
+            data-testid="export-collection-button"
+            size="xsmall"
+            onClick={openExportFileDialog}
+          >
+            Export Collection
+          </Button>
+        </div>
+        <div className={toolbarRightActionStyles}>
+          <Body data-testid="crud-document-count-display">
+            {start} - {end} of {displayedDocumentCount}
+          </Body>
+          {loadingCount && (
+            <SpinLoader size="12px" title="Fetching document count…" />
+          )}
+          {!loadingCount && (
+            <IconButton
+              aria-label="Refresh documents"
+              title="Refresh documents"
+              data-testid="refresh-documents-button"
+              onClick={onClickRefreshDocuments}
+            >
+              <Icon glyph="Refresh" />
+            </IconButton>
+          )}
+
+          <div>
+            <IconButton
+              data-testid="docs-toolbar-prev-page-btn"
+              aria-label="Previous Page"
+              title="Previous Page"
+              onClick={() => getPage(page - 1)}
+              disabled={prevButtonDisabled}
+            >
+              <Icon glyph="ChevronLeft" />
+            </IconButton>
+            <IconButton
+              data-testid="docs-toolbar-next-page-btn"
+              aria-label="Previous Page"
+              title="Previous Page"
+              onClick={() => getPage(page + 1)}
+              disabled={nextButtonDisabled}
+            >
+              <Icon glyph="ChevronRight" />
+            </IconButton>
+          </div>
+
+          <SegmentedControl
+            id={controlId}
+            aria-label="View"
+            size="small"
+            value={activeDocumentView}
+            onChange={(value) =>
+              viewSwitchHandler(value as 'List' | 'JSON' | 'Table')
+            }
+          >
+            <SegmentedControlOption
+              data-testid="toolbar-view-list"
+              aria-label="Document list"
+              value="List"
+            >
+              <Icon glyph="Menu" />
+            </SegmentedControlOption>
+            <SegmentedControlOption
+              data-testid="toolbar-view-json"
+              aria-label="E-JSON View"
+              value="JSON"
+            >
+              <Icon glyph="CurlyBraces" />
+            </SegmentedControlOption>
+            <SegmentedControlOption
+              data-testid="toolbar-view-table"
```

Comment:
Probably not the best place to report this and probably an existing bug, but I'm on the zero state and I clicked through to the table view and it shows below the zero state. Which looks a bit broken:
<img width="1904" alt="Screenshot 2022-07-27 at 10 31 57" src="https://user-images.githubusercontent.com/69737/181214174-9512866a-3be3-4282-8da7-e70d6beeab1c.png">

Reply:
Hmm actually it starts a bit low even when not on the zero state:

<img width="1904" alt="Screenshot 2022-07-27 at 10 33 51" src="https://user-images.githubusercontent.com/69737/181214497-f6697486-b66b-4bec-9db6-65cd93e80d6d.png">

Reply:
That is an existing bug, I'd like to fix that in a separate pr if that's cool.